test/Counter.t.sol

```diff
@@ -1,26 +1,12 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity 0.8.20;
 
-import "forge-std/Test.sol";
+import {Test} from "forge-std/Test.sol";
 import {Deploy} from "script/Deploy.s.sol";
-import {Counter} from "src/Counter.sol";
+import {ERC5564Announcer} from "src/ERC5564Announcer.sol";
 
-contract CounterTest is Test, Deploy {
+contract ERC5564AnnouncerTest is Test, Deploy {
```

Comment:
Should this file be removed as the code is duplicated in `ERC5564Announcer.sol`

Reply:
Ah thanks, removed in d36a4e6